### ISSUE TITLE
PyTorch Geometric

### ISSUE THREAD

**Issue by atul1234anand** [ubccr/software-layer#259]
PyTorch Geometric with CUDA and other dependencies which include pyg_lib, torch_scatter, torch_sparse, torch_cluster and torch_spline_conv

Make the installation for PyTorch 2.2 and Cuda 11.8

Link for PyTorch Geometric: https://github.com/pyg-team/pytorch_geometric

Link for additional dependencies https://data.pyg.org/whl/torch-2.2.0+cu118.html

**Comment by tonykew** [comment]
We will not be building PyTorch version 2.x until there is an easyconfig file
available for these versions of PyTorch - hopefully in the next ccrsoft software
release.

I have, however built PyTorch-Geometric version 2.2 with Coda 11.8.0 and the
current version of PyTorch, 1.13.1 - this will go through internal testing before
release.

**Comment by dsajdak** [comment]
Hi @atul1234anand pytorch-geometric has been published to the ccrsoft/2023.01 software repository:

```
[vortex:~]$ module spider geometric

----------------------------------------------------------------------------------------------------------------
  pytorch-geometric: pytorch-geometric/2.2.0-PyTorch-1.13.1-CUDA-11.8.0
----------------------------------------------------------------------------------------------------------------
    Description:
      PyTorch Geometric (PyG) is a geometric deep learning extension library for PyTorch.


    You will need to load all module(s) on any one of the lines below before the "pytorch-geometric/2.2.0-PyTorch-1.13.1-CUDA-11.8.0" module is available to load.

      gcc/11.2.0  openmpi/4.1.1

    This module provides the following extensions:

       googledrivedownloader/0.4 (E), plyfile/0.7.4 (E), python-louvain/0.16 (E), torch_cluster/1.6.0 (E), torch_geometric/2.2.0 (E), torch_scatter/2.1.2 (E), torch_sparse/0.6.18 (E), torch_spline_conv/1.2.1 (E)

    Help:

      Description
      ===========
      PyTorch Geometric (PyG) is a geometric deep learning extension library for PyTorch.


      More information
      ================
       - Homepage: https://github.com/rusty1s/pytorch_geometric


      Included extensions
      ===================
      googledrivedownloader-0.4, plyfile-0.7.4, python-louvain-0.16,
      torch_cluster-1.6.0, torch_geometric-2.2.0, torch_scatter-2.1.2,
      torch_sparse-0.6.18, torch_spline_conv-1.2.1

[vortex:~]$ module load gcc/11.2.0 openmpi/4.1.1 pytorch-geometric/2.2.0-PyTorch-1.13.1-CUDA-11.8.0
```
